### ISSUE TITLE
No preview window

### ISSUE THREAD

**Issue by stigzler** [madskristensen/MarkdownEditor2022#127]
**Describe the bug**
This used to work great. Then suddenly stopped working. I see the raw text fine, but there is no preview window, rendering this a little bit useless, really. 

**To Reproduce**
1. Right Click a .md file and select "Open With"
2. Select "MarkdownEditorV2"

**Expected behavior**
Text opens with an additional preview window. 

**Additional Info**
.md files open fine with Visual Studio Markdown Editor (but sadly this doesn't have Mermaid integration likes yours does)

**Comment by stigzler** [comment]
Guessing this extension is being mothballed given number of unanswered issues, but just in case anyone is still working on this. Still having the above issue. I've also noticed that the heading chooser at the top is missing:

![Screenshot 2024-12-06 105603](https://github.com/user-attachments/assets/3619d8a6-b824-4574-960f-cca96bcc2860)

Just wondering if this is linked? Shame this ext is no longer supported as it beats the VS built in md editor hands down (given it can display mermaid and insert TOCs).

**Comment by stigzler** [comment]
Also, latest build (from [here](https://www.vsixgallery.com/extension/MarkdownEditor2022.2347dc70-1875-4775-bf48-f2b9fdfee8d4/)) 2.0.174 has same issue.

**Comment by hsauers** [comment]
@stigzler, I had the same issue. I uninstalled and then reinstalled the extension and the preview window started working again, for me.

**Comment by stigzler** [comment]
Yeah - likewise, but then it randomly disappeared again. Shame he's no longer maintaining this as it beats the MS stock one hands down.

**Comment by grootstebozewolf** [comment]
Same issue with 2.0.183, using Visual Studio Community 2022 in folder view. After Open with (MardownEditor v2) the preview window is not shown.
